tests/helpers/stubs.py

```diff
@@ -497,37 +497,50 @@ def __init__(self):
     def list_sessions(self):
         return self.sessions
 
-
```

Comment:
As per [PEP8](https://www.python.org/dev/peps/pep-0008/), there should be two blank lines between toplevel objects (classes/functions), so this was correct - see also the [flake8 output](https://travis-ci.org/qutebrowser/qutebrowser/jobs/343390111#L477).